src/robotHexa.cpp

```diff
@@ -760,6 +763,23 @@ void RobotHexa :: transfer(ControllerDuty& controller, float duration,int transf
     ROS_INFO_STREAM("Failed to reset odometry");
   }
 
+  // reset filter
```

Comment:
Maybe something like "send a message to reset the sensor fusion filter used for odometry, if there is any" could be a little better ?